index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <!-- Meta Tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="THIS IS THE OFFICIAL WEBSITE OF YOSHITHA RATHNAYAKE PRO">
-    <meta name="author" content="YOSHITHA RATHNAYAKE PRO">
-    <meta property="og:image" content="">
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta
+      name="description"
+      content="THIS IS THE OFFICIAL WEBSITE OF YOSHITHA RATHNAYAKE PRO"
+    />
+    <meta name="author" content="YOSHITHA RATHNAYAKE PRO" />
+    <meta property="og:image" content="" />
     <!-- Title -->
     <title>YOSHITHA RATHNAYAKE PRO</title>
-    <link rel="icon" type="image/png" href="images/YOSHITHA RATHNAYAKE PRO.png" />
-</head>
-<body>
-    <h1>YOSHITHA RATHNAYAKE PRO</h1>
-    <p>This is The Official Web Site Of YOSHITHA RATHNAYAKE PRO</p>
-</body>
+    <link
+      rel="icon"
+      type="image/png"
+      href="images/YOSHITHA RATHNAYAKE PRO.png"
+    />
+  </head>
+  <body>
+    <h1 style="text-align: center">YOSHITHA RATHNAYAKE PRO</h1>
```

Comment:
Remove these inline css, just wrap the `<p>` tag with a `<center>` tag @abhi87374